guides/v2.0/contributor-guide/contributing_docs.md

```diff
@@ -53,12 +69,12 @@ To fork the devdocs repository on Github, do the following:
 
 1.  Clone the repository into your development environment and then start writing and committing your changes. Optionally, create a branch first if you plan to work on multiple changes.
```

Comment:
"Before opening a pull request, don't forget to review your work for typos, formatting errors, or sentences that need clarifying."

Reply:
Added this to the end of the list, revised content leading to it.